tests/Unit/DSCResources/DscResources.Tests.ps1

```diff
@@ -2,7 +2,8 @@ BeforeDiscovery {
     $dscResources = Get-DscResource -Module $moduleUnderTest.Name
     $here = $PSScriptRoot
 
-    $skippedDscResources = 'PowerShellRepositories', 'RemoteDesktopCollections', 'RemoteDesktopDeployment'
+    # SqlPermissions are in conflict with the Scom* resources
```

Comment:
Why @stehlih, what was the issue? The SCOM resources do not make use of SQL Permissions

Reply:
See https://github.com/nyanhp/cScom/issues/1.
The problem appears only if both resources are used in the same DSC configuration.

Reply:
Right, got it.